.github/workflows/deploy_ecs_relayer1_PROD.yml

```diff
@@ -0,0 +1,52 @@
+# The Licensed Work is (c) 2022 Sygma
+# SPDX-License-Identifier: BUSL-1.1
+
+name: Deploy ECS relayer 1
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  AWS_REGION: "us-east-2"
+  ECR_REPOSITORY: relayer-0
+  ENVIRONMENT: PROD
```

Comment:
It is not PROD it is TestNet

Reply:
Thanks, I'm using the key: `TESTNET` to make it more accurate to the environment you are expecting